cli/src/command/account.rs

```diff
@@ -627,10 +631,15 @@ pub async fn outputs_command(account: &Account) -> Result<(), Error> {
     if outputs.is_empty() {
         println_log_info!("No outputs found");
     } else {
-        let output_ids: Vec<OutputId> = outputs.iter().map(|o| o.output_id).collect();
-        println_log_info!("Outputs: {output_ids:#?}");
+        for (i, output_data) in outputs.into_iter().enumerate() {
+            println_log_info!(
+                "{i}\t{output_id}\t{output_type}",
+                i = i,
```

Comment:
Don't use names params, we never do that in the codebase, bit inconsistent

Reply:
right, fixed